server/data/helpers/refreshrHelper.js

```diff
@@ -6,16 +6,6 @@ module.exports = {
 
     return allRefreshrs;
   },
-  getClassRefreshrs: async class_id => db('refreshrs').where({ class_id }),
-  getTeacherRefreshrs: teacher_id => {
-    return db('refreshrs')
```

Comment:
I think we might need this one, or something like this, for the class create/edit page (maybe you added it somewhere else and I missed it).

Reply:
Yeah I thought these smaller ones were a bit redundant because the data can be gathered from other calls. So for example the teacher can get a list of their class, and when you call an individual class that also brings back the refreshrs, so we'd just need to strategically place our axios requests to get the data that we need

Reply:
Yeah, that totally makes sense! Some refreshrs won't be tied to classes, though, so we'd have to either keep this one or put it in the teacher request.